docs/experimental/publish-provider-command.md

```diff
@@ -0,0 +1,21 @@
+# Using the `publish-provider` command (Experimental!)
+
+## What is it?
+The publish provider command allows you to **publish** providers to a registry of your choice. Simply specify a file and the registry provider reference.
+
+## Using
+`bicep publish-provider` \<file> --target \<ref>
```

Comment:
I think it would simplify to use code blocks for the full command - that improves readability and avoids the need to escape

```suggestion
`bicep publish-provider <file> --target <ref>`
```